solr/core/src/test/org/apache/solr/cloud/api/collections/ShardSplitTest.java

```diff
@@ -76,7 +77,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-@LuceneTestCase.Nightly
+// @LuceneTestCase.Nightly
```

Comment:
Hm?

Reply:
thanks...  I finally learned how to run nightly on the command line, though not how to do it in intellij.....